README.rst

```diff
@@ -190,6 +192,34 @@ them from here: https://sourceforge.net/projects/dsgpl/files/toolkit/DSM<DSM_VER
 (e.g. https://sourceforge.net/projects/dsgpl/files/toolkit/DSM6.2)
 
 
+Testing the installation
+------------------------
```

Comment:
This is great. Could you add a sentence that explicitly states that these commands should be run on the NAS?

Reply:
Done